seminars/practice_3_lists.py

```diff
@@ -60,13 +60,17 @@ def count_evens(nums: list) -> int:
     """
     Return the number of even ints in the given array.
     """
-    # student realization goes here
+    how_many_evens = 0
```

Comment:
revert this file

Reply:
done

Reply:
no

Reply:
accidentally wrote done, the changes in the seminars prevented me from passing the checks, so I had to delete them